src/tribler/core/components/metadata_store/restapi/tests/conftest.py

```diff
@@ -13,32 +14,45 @@
 
 @pytest.fixture
 def add_fake_torrents_channels(metadata_store):
-    infohashes = []
+    def sha1_hash(value: str) -> bytes:
+        byte_array = bytearray(value, encoding='utf-8')
+        return hashlib.sha1(byte_array).digest()
 
     torrents_per_channel = 5
-    # Add a few channels
+
     with db_session:
-        for ind in range(10):
-            ext_key = default_eccrypto.generate_key('curve25519')
+        for channel_i in range(10):
+            channel_title = f'channel{channel_i}'
+            channel_hash = sha1_hash(channel_title)
+            key = default_eccrypto.generate_key('curve25519')
+
             channel = metadata_store.ChannelMetadata(
-                title='channel%d' % ind,
-                subscribed=(ind % 2 == 0),
+                title=channel_title,
+                subscribed=bool(channel_i % 2),
```

Comment:
**issue:** changing the formula for subscribed affects the resulted torrents in unexpected way, I'd revert it to:
```suggestion
                subscribed=bool(channel_i % 2 == 0),
```

Reply:
Could you give an example to prove the claim "changing the formula affects the resulted torrents in an unexpected way"?
Tests are passing.